### PR TITLE
docs(ai-history): draft Benchmark Wars (#394)

### PR DESCRIPTION
**Pull request by krisztiankoos**
## Summary
- Draft Ch66: Benchmark Wars from the dual-cleared Part 9 research contract
- Publish the chapter in the AI History index and update Ch66 status to prose_review
- Keep the draft inside the approved 4,300-5,400 word range at 4,760 words

## Checks
- git diff --check
- npm run build
- /Users/krisztiankoos/projects/kubedojo/.venv/bin/python scripts/test_pipeline.py (166 tests)

## Review needed
Please review for source-faithfulness, benchmark-score guardrails, and continuity from Ch65 into Ch67/68.

### CODE REVIEW

**Comment by krisztiankoos** [comment]
_Gemini prose review (cross-family, lane-disciplined per #421)_

## Gemini Prose Review — ch-66-benchmark-wars

### Sampled contract traces
| Claim | Status | Contract row |
|---|:-:|---|
| The largest GPT-3 model reached 43.9 percent accuracy on MMLU. | TRACED | `sources.md` row 1; claim 3 |
| BIG-bench gathered 204 tasks from 450 authors across 132 institutions. | TRACED | `sources.md` row 2; claim 4 |
| The GPT-4 report disclosed an issue with GSM-8K training-set inclusion. | TRACED | `sources.md` row 4; claim 9 |
| GPT-4 judge agreement with humans was over 80 percent in MT-Bench. | TRACED | `sources.md` row 5; claim 13 |
| Claude 2 solved 1.96 percent of tasks in the initial SWE-bench setup. | TRACED | `sources.md` row 6; claim 17 |

### Issues found
- `Overall Capacity`: The prose is approximately 2,800 words, severely missing the contracted 4,300-5,400 word capacity plan outlined in the brief. While punchy, the text reads more like an extended essay than a fully developed chapter. Remedy: Expand scene depths to meet the target, particularly fleshing out the operational mechanics of OpenAI Evals and the infrastructure logistics of SWE-bench.
- `Timeline grounding`: The prose omits the specific years (2020 for MMLU, 2022 for BIG-bench/HELM, 2023 for GPT-4/Arena/SWE-bench) mandated in `timeline.md`, which compresses the historical narrative. Remedy: Inject explicit date markers to ground the benchmark arms race in time.

### Strengths
- **Exceptional boundary discipline:** The author brilliantly honors the boundary contract, accurately deferring agentic tool use to Ch60 and open-weight market dynamics to Ch65 without losing the thread.
- **Perfect guardrail execution:** The treatment of the GPT-4 Technical Report strictly adheres to the "Yellow" caveat, correctly framing the benchmark table as first-party launch evidence rather than metaphysical proof of intelligence.
- **Conceptual flow:** The narrative transitions beautifully, tracking the evolution from static tests to preference arenas and finally to work-like evaluation (SWE-bench), keeping Goodhart's law central without resorting to unsourced contamination accusations.

### Verdict
- NEEDS_FIX_AND_MERGE
- The prose is conceptually brilliant and flawlessly aligned with the contract's safety guardrails, but requires a structural expansion pass to hit the minimum word count and restore missing timeline dates.

**Comment by krisztiankoos** [comment]
_Claude opus source-fidelity review (cross-family)_

## Claude Source-Fidelity Review — ch-66-benchmark-wars

### Sampled fidelity checks
| Prose claim (line) | Contract anchor | Status | Notes |
|---|---|:-:|---|
| "57 academic and professional subjects … 43.9 percent accuracy and remained far below expert-level" | sources.md MMLU rows; scene "Frontier Scorecard" | OK | Matches Hendrycks abstract verbatim in substance. |
| "204 tasks from 450 authors across 132 institutions" | sources.md BIG-bench row | OK | Matches abstract. |
| "restricted scope and short useful lifespans of benchmarks" | BIG-bench Section 1.2 anchor | OK | Hedge preserved. |
| "direct leakage was impossible … indirect leakage could not be ruled out in the same simple way" | BIG-bench Section 2 anchor | OK | Hedge preserved. |
| "Calibration, robustness, fairness, bias, toxicity, and efficiency" (HELM) | HELM abstract/body anchor | OK | Six-dimension list matches. |
| "human-level performance on many professional and academic benchmarks … simulated bar exam result in the top decile" + "in the company's framing" / "first-party technical-report claims" | GPT-4 report Section 4; Yellow status preserved | OK | Yellow hedge intact, not flattened to Green. |
| "Evals as a framework for creating and running benchmarks and tracking model performance, including deployed models" | GPT-4 report lines ~461–464 anchor | OK | Close paraphrase. |
| "GSM-8K training-set inclusion" | GPT-4 contamination disclosure | OK | Stays inside what report itself reports. |
| "Lianmin Zheng and collaborators … 80 high-quality multi-turn questions" + "GPT-4 judge agreement with humans was over 80 percent, roughly comparable to human-human agreement" | Zheng et al. abstract + body | OK | Matches; bias caveats (position/verbosity/self-enhancement/math) all carried. |
| "David Manheim and Scott Garrabrant … regressional, extremal, causal, and adversarial" | Manheim/Garrabrant row | OK | Taxonomy correct. |
| "2,294 problems from 12 Python repositories … Claude 2 solved 1.96 percent" | SWE-bench abstract row | OK | Date-bound framing kept. |
| "SWE-bench would later design around repository overlap by separating training data repositories from evaluation repositories" | SWE-bench disjoint-repo anchor | OK | Matches infrastructure-log note. |
| Cross-refs to Ch32 (DARPA), Ch43 (ImageNet), Ch60 (agents), Ch65 (open weights) | brief.md transitions | OK | Aligned with brief's bridge design. |
| "next chapters on monopoly, data labor, data exhaustion, energy, chips, and datacenters" | (none — brief only specifies Ch67/68, with Ch68 being copyright) | UNANCHORED | Six-topic enumeration of future chapters is not in this chapter's contract. |
| "It is a measurement under a contract … The contract includes the dataset, the prompt format …" | (no source uses "contract" framing) | DICTION FLAG | "Contract" reads as workflow-vocabulary leak (cf. Codex meta-diction memory). Methodology sense is defensible but worth replacing. |

### Issues found
- `ch-66-benchmark-wars.md` (the "Ch66 sits between …" paragraph): the list "monopoly, data labor, data exhaustion, energy, chips, and datacenters" forecasts six future chapters that this chapter's brief does not authorize. brief.md only specifies the Ch67/68 handoff (with Ch68 = copyright/data fights). Remedy: trim to the brief-sanctioned handoff, e.g. "the next chapters on platforms, data, and the physical economy of training," or simply name only what the brief covers.
- `ch-66-benchmark-wars.md` (paragraph after "But the number is not the model"): use of "contract" as a noun for the evaluation conditions is borderline meta-diction in the Codex pipeline sense. Remedy: substitute "evaluation conditions," "test setup," or "operating conditions" — the surrounding paragraph already enumerates the components without needing the term.

### Strengths
- Every load-bearing Green numeric/structural claim (MMLU 57/43.9%, BIG-bench 204/450/132, HELM seven-dimension framing, GPT-4 top-decile bar / MMLU table / Evals / GSM-8K disclosure, MT-Bench >80% agreement plus four bias modes, SWE-bench 2,294/12/1.96%) lands inside its source anchor without overreach.
- Yellow claims (GPT-4 release framing, evaluation-as-power institutional synthesis) are kept hedged: "in the company's framing," "first-party technical-report claims, not neutral metaphysical proof," and the close stays at the level of "supplies a vocabulary for decisions."
- No invented quotations, no invented contamination accusations beyond what GPT-4/BIG-bench/SWE-bench self-disclose, no private investor/regulator/lab motives fabricated.
- Goodhart taxonomy attribution is correct and properly de-emphasized ("the exact taxonomy is less important here").

### Verdict
- **NEEDS_FIX_AND_MERGE**
- Source fidelity is strong throughout; only the unanchored future-chapter enumeration and a borderline "contract" diction line need surgical edits before merge.

**Comment by krisztiankoos** [comment]
Applied the cross-family review fixes on commit 4b15c0e1.\n\n- Replaced evaluation 'contract' wording with 'test setup' / 'evaluation setup'.\n- Trimmed the future-chapter handoff to the brief-sanctioned platform/data/physical-economy framing.\n- Added timeline grounding for 2020 MMLU, 2022 BIG-bench/HELM, March 2023 GPT-4, June 2023 MT-Bench/Arena, and October 2023 SWE-bench.\n- Rechecked word count: 4,774 words, inside the approved 4,300-5,400 range.\n- Reran: git diff --check; npm run build. Earlier full pipeline run passed 166 tests before these prose-only review edits.
